autoload/ctrlp.vim

```diff
@@ -428,7 +428,8 @@ endf
 if has('patch-8.2-0995')
 	fu! s:GlobPath(dirs, depth)
 		let entries = []
-		for e in split(a:dirs, ',')
+		let dirs = substitute(a:dirs, '\\\([%#]\)', '\1', 'g')
```

Comment:
This doesn't fix #138.
The second argument should be `'\\\([%# ]\)'`. (white-space is missing here)
A patch you commented on a ticket #138 is correct, but this MR doesn't contain that.

Reply:
Right. I'll fix.